src/Tasks/CreateVisualBasicManifestResourceName.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.Build.Tasks
     /// </summary>
     public class CreateVisualBasicManifestResourceName : CreateManifestResourceName
     {
+        public override string SourceFileExtension => ".vb";
```

Comment:
Is there an F# equivalent to this somewhere? cc @kevinransom

Reply:
Looks like there is: https://github.com/dotnet/fsharp/blob/5f35d4e653d39bf5a5d8a18ac576f97eabe6c368/src/fsharp/FSharp.Build/CreateFSharpManifestResourceName.fs